main.scss

```diff
@@ -0,0 +1,18 @@
+@import 'o-colors/main';
+@import 'o-typography/main';
+@import 'o-grid/main';
+@import 'o-icons/main';
+
+@import 'scss/variables';
+@import 'scss/mixins';
+
+@import 'scss/elements';
+@import 'scss/layout';
+@import 'scss/themes';
+
+@if ($o-topper-is-silent == false) {
+	@include oTopperElements;
+	@include oTopperColors;
```

Comment:
Any reason themes aren't output by default?

Reply:
I imagine if you only want a couple of themes it would be better to stick to non-silent mode.

Reply:
I'm following the lead of next-article here, which splits themes out into their own stylesheet. Themes are about 50% of the CSS weight, and are used in very few articles